rest/rest-sqs/src/main/scala/org/elasticmq/rest/stats/package.scala

```diff
@@ -0,0 +1,6 @@
+package org.elasticmq.rest
+
+package object stats {
+  type Attributes = Map[String, String]
```

Comment:
is this used?

Reply:
Yes, in case-class returned in rest endpoint.

Reply:
ok :)